src/spec-lang/tc/internal_types/import_ref_type.ts

```diff
@@ -0,0 +1,20 @@
+import { assert, ImportDirective, TypeNode } from "solc-typed-ast";
+import { Range } from "../../ast";
+
+export class ImportRefType extends TypeNode {
```

Comment:
@blitz-1306 mentioned that perhaps we should migrate this eventually to solc-typed-ast

Reply:
Will make a note of this cleanup action.